utils/benchmark/setupForTests.sh

```diff
@@ -35,32 +42,28 @@ done
 
 # Install @ima scoped packages from local registry
 npm config set @ima:registry=$NPM_LOCAL_REGISTRY_URL
+npm config set legacy-peer-deps true
 
 # Update create-ima-app versions
-cd "$ROOT_DIR"
+cd "$ROOT_DIR_IMA"
 node utils/version/create-ima-app-versions.js
 # Link current create-ima-app version to global scope
 cd "$CREATE_IMA_APP_DIR"
 npm link
 
 # Setup app from example hello
-cd "$ROOT_DIR"
+cd "$ROOT_DIR_IMA"
```

Comment:
If you switch this back to `$ROOT_DIR`, then you dont need to do the `mv "$ROOT_DIR_IMA/ima-app" "$ROOT_DIR"` later.

Reply:
Use `$ROOT_DIR` again.